File: Level1/5.2016년/5.2016년.py
def solution(a, b):
    answer = ''
    week = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"]
    sumDay = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    count = 0
    for i in range(0,a-1):
        count += sumDay[i]
    count += b-1
    answer = week[(count+5)%7]
    return answer

File: Level1/5.2016년/5.2016년.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int a, int b) {
    string answer = "";
    string week[] = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};
    int sumDay[] = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
    int count = 0;
    for(int i = 0; i<a-1 ; i++){
        count += sumDay[i];
    }
    count += b-1;
    answer = week[(count+5)%7];
    return answer;
}